region/crateria/central/Landing Site.json

```diff
@@ -1268,9 +1283,10 @@
     {
       "id": 55,
       "link": [4, 3],
-      "name": "SpeedyJump Into SpringballJump",
+      "name": "Tricky HiJump SpringBall Jump",
       "requires": [
-        "canTrickyDashJump",
+        "canTrickyJump",
```

Comment:
Moved this down to VH. Do you think it should be moved to Hard?

Reply:
Seems mechanically fine for hard, very similar to the movement to reach pantry item.
I don't expect people to do it, but it could tighten health counts to where they start thinking about it.